docs/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ nav_order: 5
 
 ## main
 
+* Add source_code_uri to gemspec.
```

Comment:
```suggestion
* Add `source_code_uri` to gemspec.
```

Reply:
@Spone Thanks for your suggestion :heart: